daemon/daemon_main.go

```diff
@@ -740,6 +740,10 @@ func init() {
 	flags.Int(option.EndpointQueueSize, defaults.EndpointQueueSize, "size of EventQueue per-endpoint")
 	option.BindEnv(option.EndpointQueueSize)
 
+	flags.Duration(option.EndpointGCInterval, 5*time.Minute, "Periodically monitor local endpoint health via link and netns status on this interval and garbage collect them if they become unhealthy, set to 0 to disable")
```

Comment:
Nit: There's no netns logic specifically AFAICT, so do we need to mention this?

Reply:
:+1: I can drop the netns bit.

I was debating how much detail to put here. On the one hand, if it's vague then it's hard for users to interpret the meaning. On the other hand if it's too specific, it could be wrong if we ended up updating the logic in future for some reason.